themes/vocabulary_theme/templates/layout.html

```diff
@@ -8,6 +8,9 @@
 <link rel="apple-touch-icon" sizes="180x180" href="{{ '/static/vocabulary/favicon/apple-touch-icon.png' |url}}" />
 <link rel="stylesheet" href="{{ '/static/css/style.css'|url }}">
 <link rel="stylesheet" href="{{ get_pygments_stylesheet()|url }}">
+<link rel="stylesheet" href="https://cdn.datatables.net/2.2.1/css/dataTables.dataTables.css" />
```

Comment:
we should localize this to the repository with a `vendor` directory (see how vocabulary handles the normalize css file), and not pull in via a cdn.

Reply:
we also should ONLY load this on pages that are using data tables, rather than having it loaded for every page.